src/plugins/action_change_status/src/changeStatus.ts

```diff
@@ -2,19 +2,38 @@ import { ActionPluginOutput, Status } from '../../types'
 
 type IParameters = {
   applicationId: number
+  reviewId: number
   newStatus: Status
 }
 
 module.exports['changeStatus'] = async function (
   parameters: IParameters,
   DBConnect: any
 ): Promise<ActionPluginOutput> {
-  const { applicationId, newStatus } = parameters
+  const { applicationId, reviewId, newStatus } = parameters
+
+  if (applicationId) {
+    return await changeApplicationStatus(applicationId, newStatus, DBConnect)
+  } else if (reviewId) {
```

Comment:
Haven't tested yet either, just making a couple of quick comments:
`if (applicationId)` -- if you're wanting to change the review status, you'll still get an applicationId passed in (I think), so not sure this is a good way to determine whether we're updating review status or application status.

2 other ways I can think of:
- check the Trigger
- add a new parameter and explicitly declare it in the Action definition of the template.

Reply:
Okay, have tested now. I see how it works, but the Action definition in the template query had some errors, have made suggestions below, and it's working now.

Reply:
Thanks for fixing